samples/service.js

```diff
@@ -24,30 +34,60 @@ function createService (config) {
   }
 
   function readSample (id) {
-    const path = join(samplesDirectory, id)
+    const path = _getSamplePath(id)
 
     return fs.readFile(path)
-      .then(({ buffer }) => {
+      .then(data => {
+        const { buffer } = data
         return audioContext.decodeAudioData(buffer)
+          .then(audioBuffer => ({ data, audioBuffer }))
       })
-      .then(audioBuffer => ({ id, path, audioBuffer }))
+      .then(({ data, audioBuffer }) => ({ id, path, data, audioBuffer }))
```

Comment:
`analyzeSample` uses this function, and wants the `data` as well. including here saves us from another expensive read operation